Remove the register-cert step from Getting Started with Scalar DL Sandbox (#3)
scalar-labs/scalardl: docs/sandbox.md

The access token is only used for authentication with Sandbox API gateway.
The key pair is used for communicating with Scalar DL network.

> Please note that we generate a pair of a private key and a certificate for ease of use for the Sandbox environment, but it is usually required to create the pair in your own environment.

Reviewer: ```suggestion
Please note that we generate a pair of a private key and a certificate and register the certificate for ease of use for the Sandbox environment, but those steps are usually required in your own environment.
```

> The generated certificate is already registered to Scalar DL in the Sandbox environment, so you don't have to take a step to register it.

Reviewer: I updated the sentence above, so I think we can delete this line.
